app/controllers/db.js

```diff
@@ -0,0 +1,28 @@
+
+var models = require('../../models');
+var Brother = models.Brother;
+
+module.exports = {
+  sign_in: function(id, name, email) {
+    Brother.findOrCreate({ 
+      where: {id: id},
+      defaults: {
+        firstName: name.givenName,
+        lastName: name.familyName,
+        email: email[0].value
+      }
+    }).then(function() {});
+  },
+  findUser: function(id) {
+    return Brother.findById(id);
+  },
+  updateUser: function(user, id) {
+    return Brother.update(user,
+      {
+        where: { id: id },
+        fields: ['pin']
```

Comment:
via an endpoint, yes. this will be expanded to edit position buy i dont want the user to be able to edit their own name or email or anything